src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -12,13 +12,24 @@ import useCreateChat from '../../../hooks/screen/useCreateChat';
 import TokenStorage, {ITokenEnum} from '../../../utils/storage/custom/tokenStorage';
 import {Context} from '../../../context';
 import {checkUserBlock} from '../../../service/profile';
-import {getChatName, getOfficialAnonUsername} from '../../../utils/string/StringUtils';
+import {
+  getChannelListInfo,
+  getChatName,
+  getOfficialAnonUsername
+} from '../../../utils/string/StringUtils';
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import {setChannel} from '../../../context/actions/setChannel';
 import {setParticipants} from '../../../context/actions/groupChat';
+import {addMemberGroup, removeMemberGroup, leaveGroup} from '../../../service/chat';
+import UserSchema from '../../../database/schema/UserSchema';
+import ChannelListSchema from '../../../database/schema/ChannelListSchema';
+import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import useUserAuthHook from '../../../hooks/core/auth/useUserAuthHook';
+import useChatUtilsHook from '../../../hooks/core/chat/useChatUtilsHook';
 
 const useGroupInfo = (channelId = null) => {
   const navigation = useNavigation();
+  const {localDb, refresh} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to include new functions and components. The changes seem appropriate as they are necessary for the new functionality being added.
```diff
+ import {
+   getChannelListInfo,
+   getChatName,
+   getOfficialAnonUsername
+ } from '../../../utils/string/StringUtils';
+ import {addMemberGroup, removeMemberGroup, leaveGroup} from '../../../service/chat';
+ import UserSchema from '../../../database/schema/UserSchema';
+ import ChannelListSchema from '../../../database/schema/ChannelListSchema';
+ import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+ import useUserAuthHook from '../../../hooks/core/auth/useUserAuthHook';
+ import useChatUtilsHook from '../../../hooks/core/chat/useChatUtilsHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->